### PR TITLE
Add support for multiple masters to prometheus

### PR DESCRIPTION
**Pull request by mborsz**
/assign @mm4tt

### CODE REVIEW

File: clusterloader2/pkg/prometheus/prometheus.go [hunk 1]
```diff
@@ -227,10 +227,10 @@ func dumpAdditionalLogsOnPrometheusSetupFailure(k8sClient kubernetes.Interface)
 	klog.Info(string(s))
 }
 
-func getMasterIp(clusterConfig config.ClusterConfig) (string, error) {
-	if clusterConfig.GetMasterInternalIp() != "" {
-		klog.Infof("Using internal master ip (%s) to monitor master's components", clusterConfig.GetMasterInternalIp())
-		return clusterConfig.GetMasterInternalIp(), nil
+func getMasterIps(clusterConfig config.ClusterConfig) ([]string, error) {
+	if len(clusterConfig.MasterInternalIPs) != 0 {
+		klog.Infof("Using internal master ip (%s) to monitor master's components", clusterConfig.MasterInternalIPs)
```
Comment (wojtek-t) [inline clusterloader2/pkg/prometheus/prometheus.go]: nit:
s/ip/ips/

**Comment by mborsz** [comment]
PTAL

**Comment by mm4tt** [comment]
/approve

Nice!

**Comment by wojtek-t** [comment]
LGTM

**Comment by mm4tt** [comment]
Nice!

**Pull request merged**